Neos.Neos/Classes/Controller/Frontend/NodeController.php

```diff
@@ -326,22 +322,24 @@ private function fillCacheWithContentNodes(
             self::fillCacheInternal(
                 $childSubtree,
                 $currentDocumentNode,
-                $inMemoryCache
+                $this->subgraphCachePool,
+                $subgraph
```

Comment:
This `$subgraph` is the *uncached*  version. IMO we should use the cached version here

Reply:
done